docs/README.md

```diff
@@ -42,23 +42,21 @@ Following sections are Kubernetes specific. If you are Kubernetes user, use foll
 * Dynamic provisioning - uses persistence volume claim (PVC) to let the Kuberenetes to create the FSx for Lustre filesystem for you and consumes the volume from inside container.
 
 ### Installation
-Checkout the project:
-```sh
->> git clone https://github.com/aws/aws-fsx-csi-driver.git
->> cd aws-fsx-csi-driver
-```
-
-Edit the [secret manifest](../deploy/kubernetes/secret.yaml) using your favorite text editor. The secret should have enough permission to create FSx for Lustre filesystem. Then deploy the secret:
+#### Set up driver permission
+The driver requires IAM permission to talk to Amazon FSx for Lustre service to create/delete the filesystem on user's behalf. There are several methods to grant driver IAM permission:
```

Comment:
Is one prefered over another? Should we mention eventual down sides of the different methods?

Reply:
Pod identity will be our ultimate solution. At this point, I don't have too much preference over one or the other. How about you?

Reply:
Both sound fine to me at the moment. 🙂